.c3i/authorized_users.yml

```
@@ -1336,3 +1336,10 @@ authorized_users:
 - pierricgimmig
 - laci-aura
 - BenBudr
+- PLeVasseur
+- datalogics-staylor
+- soroosh-sdi
+- TheHugeManatee
+- FannoFlow1
+- nayakche
+- psi-func
```

```suggestion
- psi-func
- deniskovalchuk
```

Added deniskovalchuk from last minute